pkg/abi/linux/fuse.go

```diff
@@ -246,3 +246,58 @@ type FUSEInitOut struct {
 
 	_ [8]uint32
 }
+
+// FUSEGetAttrIn is the request sent by the kernel to the daemon,
+// to get the attribute of a inode
```

Comment:
It would be a good idea to use `.` consistently: if the comments inside the struct uses `.`, perhaps it would be better to use one here as well.

Same for others